### PR TITLE
Issue #10548: Migrate to Inline Config Parser in AvoidDoubleBraceInitializationCheckTest

### PR DESCRIPTION
**Pull request by shashwatj07**
issue #10548

### CODE REVIEW

**Review by romani — CHANGES_REQUESTED** [review]
Items: 
Please apply to all other PRs

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java [hunk 1]
```diff
@@ -11,12 +11,12 @@
 
 public class InputAvoidDoubleBraceInitialization {
     void m() {
-        ArrayList<?> list = new ArrayList<Object>() { // violation
+        ArrayList<?> list = new ArrayList<Object>() { // violation 'Avoid double brace initialization.'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java:14]: Please do not use any messages when Check has single message https://checkstyle.org/config_coding.html#AvoidDoubleBraceInitialization_Violation_Messages

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java (hunk 1 above)
Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java]: done

**Review by timurt — APPROVED** [review]

**Review by romani — APPROVED** [review]

**Pull request merged**